src/OrchestratorEndpoint.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\App\OrchestratorTrigger;
+
+use Keboola\Component\UserException;
+use Keboola\StorageApi\Client as StorageApiClient;
+use Keboola\StorageApi\ClientException;
+
+class OrchestratorEndpoint
+{
+    public const ORCHESTRATOR_COMPONENT_ID = 'orchestrator';
+
+    public static function detect(string $kbcToken, string $kbcUrl): string
+    {
+        $sapiClient = new StorageApiClient([
+            'token' => $kbcToken,
+            'url' => $kbcUrl,
+            'backoffMaxTries' => 3,
+        ]);
+
+        try {
+            $index = $sapiClient->indexAction();
+            foreach ($index['components'] as $component) {
+                if ($component['id'] !== self::ORCHESTRATOR_COMPONENT_ID) {
```

Comment:
nebylo by to čitelnější opačně? tj. return když se to rovná?

Reply:
Nevim o tom ze bych zadal o review :-)

Reply:
jj, já se snažím číst všechno průběžně, když na to narazím, abych se do co nejvíc věcí dostal. A když už na to koukám a něco najdu... ;)

Reply:
jsem to upravil